### ISSUE TITLE
Updated Questions 

### ISSUE THREAD

**Issue by ErinCain** [FlowWest/edi-feather-snorkel#5]
-    Decided to change Mo's Ditch for unit 28 being consistent with map, but not slides (no Mo's Ditch, but located in "Hatchery Ditch). 
                - Is there a mos ditch? if so, what units are within it? 
                - Does section 3 = mo's ditch? 
- Is it okay to rename turbidity = visibility in the early data? If these are not the same is there a way we can convert. 
- If "units_covered" is blank does that mean all units within the section are covered? If so then maybe we could fill that in based on the lookup.
- Section names...help with all the messy ones.
- Is it possible that "V" is getting confused for something else when encoded for instream cover? 
- Is it okay to filter below 6 messy units out. If not, how should we handle? 
```
filter(!unit %in% c("77-80", "86-89",
                      "104, 106, 112", "104 106  112",
                      "104 106 112", "446/449"), # filtered out these messy units for now, alternatively we can see if casey can assign a non messy unit
```
We were able to associate a river mile (from the "SnorkelSections_RiverMiles" lookup table) with every unit except the following - `"HGSC"  "HPRL"  "CF"    "BRB"   "HTPSC" "UHSC"  "SEC2"  "RBUB" & "77-80"         "86-89"         "104, 106, 112" "104 106  112"  "104 106 112"   "446/449"       "543"  `.  For any unit that does not fall into a section (as described in [these slides](https://netorg629193.sharepoint.com/:p:/s/VA-FeatherRiver/EczBk-i0VfhFn223jr6AmbYBxG1Aw8W4ZfIs3Khg9R2wgg?e=Uhd9bR)) we are planning to assume that the best spatial resolution we can get on that unit is river mile. Is there any other info we should be using instead?

**Comment by alvizek** [comment]
Response from Casey

I think we are getting really close on the Feather River Snorkel Survey data package. We just had a few additional questions that we wanted to wrap up before sending the package for review.

- We are unclear on the site designation of Mo’s Ditch and Hatchery Ditch. It looks like unit 28 is in Hatchery Ditch but it was labeled Mo’s Ditch in the data. Are Hatchery Ditch and Mo’s Ditch the same thing? If not, what units are within Mo’s Ditch (we didn’t see that one on the slides)?

Answer: Good question, I see how that could be confusing. They are not the same. Moe’s Ditch (unit 28) is on the Hatchery Ditch slide in the PowerPoint.

- Is it okay to rename turbidity = visibility in the early data? If these are not the same is there a way that we can convert visibility to turbidity?

Answer: We currently measure visibility by using a lure with parr marks and seeing how far away a diver can distinguish the parr marks. If turbidity was collected in the earlier data using a turbidimeter then we don’t have a way to convert to a measure of visibility.

- If "units_covered" is blank does that mean all units within the section are covered?

Answer: Unfortunately, I think we would have to assume yes that all units were covered.

- We noticed a few places where “V” was added as a code for instream cover. Is it possible that "V" is getting confused for something else when encoded for instream cover?

Answer: I have no idea what that could be referring. If I had to guess “vegetation”?

- Is it okay to filter below 6 messy units out. If not, how should we handle these?
         filter(!unit %in% c("77-80", "86-89",
                      "104, 106, 112", "104 106  112",
                      "104 106 112", "446/449"), 

Answer: I’m not sure where these are occurring in the data.  

- We were able to associate a river mile (from the "SnorkelSections_RiverMiles" lookup table) with every unit except the following:  "HGSC" "HPRL" "CF" "BRB" "HTPSC" "UHSC" "SEC2" "RBUB, "543". Are you able to map any of these to an existing unit? For any unit that does not fall into a section (as described in [these slides](https://urldefense.proofpoint.com/v2/url?u=https-3A__netorg629193.sharepoint.com_-3Ap-3A_s_VA-2DFeatherRiver_EczBk-2Di0VfhFn223jr6AmbYBxG1Aw8W4ZfIs3Khg9R2wgg-3Fe-3DUhd9bR&d=DwMGaQ&c=euGZstcaTDllvimEN8b7jXrwqOf-v5A_CdpgnVfiiMM&r=xhNB0udPC7hKQ3VgH-cILvMy09D0YDWZorx6_jPelS4&m=jC5M2FAR765Fh06GO0t1Eo-ZchmcG-N1Zj9eqfPepxDnAGeSsu-9PHsYw_cL2KQE&s=c3vKxPNZvwurvhNbi1IXu0WWM07ygxWIJlUKHlW9M1g&e=)) we are planning to assume that the best spatial resolution we can get on that unit is river mile. Is there any other info we should be using instead?

Answer: These are going to be sites that were surveyed during high flow when the crew determined it was not safe enough or meaningful to snorkel the regular sites.  I’ve provided a .kmz that shows what I imagine the center point to be. It would be hard to bound those locations like we do with the regularly surveyed units. I can’t find any record of “543” in the DB.
